bit/base58.py

```diff
@@ -49,8 +49,9 @@ def b58decode(string):
             num *= 58
             num += alphabet_index[char]
     except KeyError:
-        raise ValueError('"{}" is an invalid base58 encoded '
-                         'character.'.format(char)) from None
+        raise ValueError(
+            '"{}" is an invalid base58 encoded ' 'character.'.format(char)
```

Comment:
```suggestion
            '"{}" is an invalid base58 encoded character.'.format(char)
```

Reply:
Thanks for pointing that out. Really weird.